src/test/java/com/revature/controllertests/CurriculumControllerTest.java

```diff
@@ -0,0 +1,10 @@
+package com.revature.controllertests;
+
+import org.springframework.boot.test.context.SpringBootTest;
+
+import com.revature.cmsforce.CMSforceApplication;
+
+@SpringBootTest(classes = CMSforceApplication.class)
```

Comment:
I'd recommend excluding this and creating a branch off for testing - that way we can get these changes in sooner